### PR TITLE
Splash text is added to the actual splash screen now

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
I updated my splash text module to also add the text to the title screen itself, close to the beestation text. (this is defined per-splash, default is 0,0)

## Why It's Good For The Game
it looks cool and minecraft esque, little bitta comedy

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: splash text is now added to the title
/:cl:

### CODE REVIEW

**Comment by DrDuckedGoose** [comment]
This is an issue that can occur. 
You might want to consider the line length into the offset.
image

**Comment by etherware-novice** [comment]
ooo yea..do you think doing x -= length(splash) would fix that?

**Comment by DrDuckedGoose** [comment]
> ooo yea..do you think doing x -= length(splash) would fix that?

maybe `` x -= 10*length(splash)`` since each letter is bigger than 1 pixel. You can find the exact width of the font and substitute that for ``10``.

**Comment by PowerfulBacon** [comment]
Should be slightly tiled and animated.
Would be super nice if we could have a countdown, goon-style

File: strings/splash.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
+		"The icon of anticipation.",
 		"The only place where you can hit yourself with a toolbox on accident.",
-		"Where the Chain of Command is a Chain of Incompetence",
+		"Where the Chain of Command is a Chain of Incompetence.",
 		"AI LAW 2, START THE GAME",
 		"Honk Honk! You should pick clown.",
 		"Watch for nukies!",
 		"Traitor Central!",
-		"You've definitely seen this before.",
+		"Sponsored by space cola...soon.",
```
Comment (itsmeow) [inline strings/splash.json:14]: what

File: code/game/turfs/closed/_closed.dm [hunk 2]
```diff
@@ -69,12 +69,28 @@ INITIALIZE_IMMEDIATE(/turf/closed/indestructible/splashscreen)
...
+
+/turf/closed/indestructible/splashscreen/proc/update_splash()
+	var/splash_text = pick(strings(SPLASH_DESC_FILE, "splashes"))
+	maptext = MAPTEXT("<font color='ebfc50'>[splash_text]</font>")
+	maptext_height = world.icon_size * 4
+	maptext_width = world.icon_size * 16
+	var/list/offsets = get_title_offset()
+	maptext_x = offsets[1]
+	maptext_y = offsets[2]
+
```
Comment (itsmeow) [inline code/game/turfs/closed/_closed.dm:83]: rotate the maptext here 45 degrees
use animate() to make it get slightly bigger/smaller like minecraft

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/game/turfs/closed/_closed.dm (hunk 2 above)
Comment (etherware-novice) [reply code/game/turfs/closed/_closed.dm:83]: how do i rotate and how do i animate it

Comment (PowerfulBacon) [reply code/game/turfs/closed/_closed.dm:83]: Rotate is a matrix proc and animate is just animate(). Check the byond documentation for more details.
If you have the DM language client installed in VS code, you can control click on byond functions to view their documentation, or you can find the docs online. They are basic, but informative.

Comment (etherware-novice) [reply code/game/turfs/closed/_closed.dm:83]: well, would that work with maptext is the problem,

Comment (PowerfulBacon) [reply code/game/turfs/closed/_closed.dm:83]: You would have to rotate whatever the maptext is attached to, which would require a holder effect object for the text

Comment (etherware-novice) [reply code/game/turfs/closed/_closed.dm:83]: holder effect object?

Comment (etherware-novice) [reply code/game/turfs/closed/_closed.dm:83]: can it just be flat?

**Pull request closed without merging**